pkg/binding/format/doc.go

```diff
@@ -0,0 +1,8 @@
+package format
+
+/*
+Package format formats structured events.
```

Comment:
only structured?

Reply:
yes, this package is intended to provide the primitives to decode/encode messages in structured mode